src/main/docker/jenkins/Dockerfile

```diff
@@ -1,54 +1,15 @@
 FROM jenkins/jenkins:lts
 
-LABEL description="Jenkins with maven, python3.6 and gcc libraries pre-installed for Artemis"
+LABEL description="Jenkins with plugins pre-installed for Artemis"
 
 USER root
 
-RUN apt update
```

Comment:
Why do you remove all of these steps?
We need to at least install Java here, as Jenkins only ships with JDK 8/11.
Also removing C and Python without a replacement would also not be that smart IMO.

I am currently cooperating with Henning to apply the same changes that I did to the Atlassian setup to this setup. Meaning the Dockerfiles would be extracted into a separate Repository. I also got both the C and the Python installations working already.

Reply:
Since Artemis builds using docker, you don't need dependencies here. Jenkins simply needs the ability to run containers.

Reply:
https://github.com/ls1intum/Artemis/pull/4407#discussion_r783911848

Reply:
Makes sense. Thanks for the reply.